src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div>
+        <img
+          src=".//images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        >
+      </div>
```

Comment:
This div is redundant
```suggestion
      <img
        src=".//images/imac.jpeg"
        alt="imac"
        class="card__image"
      >
```